stacks/bracketstester.js

```diff
@@ -0,0 +1,30 @@
+const Stack = require('./Stack');
+
+function BracketsTester(stackMaxCapacity) {
+    this.stringStack = new Stack(stackMaxCapacity);
+    this.count = 0;
+}
+
+BracketsTester.prototype.checkBrackets = function(stringValue) {
+
+    Array.from(stringValue).forEach(charValue => this.stringStack.push(charValue));
```

Comment:
what if you only pushed brackets in the stacks, would have saved time when you started verifying it.

Reply:
good idea, now adding only brackets